src/About.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import './App.css';
+
+class App extends Component {
+  render() {
+    return (
+      <body>
```

Comment:
Each component should NOT have <body> and <body/> tags. There should only be one set per page and they are already in public/index.html.  index.html is the only page we ever have in a Single Page Application (SPA). As default this is what create-react-app creates for us.